Sources/Contour/Configurator/EndpointElement.cs

```diff
@@ -175,5 +177,21 @@ public ValidatorCollection Validators
             }
         }
 
+        [ConfigurationProperty("excluded-headers")]
+        public string ExcludedHeadersString => (string)base["excluded-headers"];
+
+        public IEnumerable<string> ExcludedHeaders
```

Comment:
This type is not responsible for string operations. It should only read the configuration. This should be moved to a place where a list of headers is obtained from the configuration property.